<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox{
	border-radius: 3px;
	padding: 1px 10px 1px 5px;
	border: 1px solid #636363;
}
QComboBox:hover{
	border-radius: 3px;
	padding: 1px 10px 1px 5px;
	border: 1px solid #636363;
	
	background-color: rgb(170, 255, 255);
}
QComboBox::drop-down{
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px; 
	border-left-width: 1px;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
	border-left-color: #636363;
}

QComboBox::down-arrow{
	image: url(:/image/array_down.png); 
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>C0</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C3</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item row="4" column="3">
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item row="5" column="3">
       <property name="text">
        <string>24</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 126, 126);
color: rgb(255, 255, 255);
font: 75 12pt &quot;Consolas&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>aaaaa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>bbbb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ccccc</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dddd</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
